React/Fabric/Mounting/ComponentViews/Modal/RCTModalHostViewComponentView.mm

```diff
@@ -180,6 +183,28 @@ - (void)ensurePresentedOnlyIfNeeded
   }
 }
 
+- (void)handleDetent:(ModalHostViewDetent)detent
+{
+  if (@available(iOS 15.0, *)) {
+    if (self.viewController.sheetPresentationController) {
```

Comment:
I would early exit here, to avoid too much indentation.
```suggestion
    if (!self.viewController.sheetPresentationController) { return; }
```

Reply:
Thanks this makes sense :)